server/integration/run_and_assert_pokemon_app_test.go

```diff
@@ -29,4 +29,5 @@ func TestExecutorIntegration(t *testing.T) {
 	}
 
 	assert.Equal(t, 2, count)
+	assert.Greater(t, len(testRun.Trace.Flat), 8)
```

Comment:
I left it as 8 because we have some spans that sometimes show up and other times don't show up at all (`SET statements in the database`). So, if none of them show up in the trace, we have `9` spans. If they show up, we get `11`